### ISSUE TITLE
Out of date for current version 1.4.2 of Seahorse

### ISSUE THREAD

**Issue by rabejens** [deepsense-io/seahorse-sdk-example#1]
The master cannot be built because `seahorse-executor-deeplang` is not present on the repositories, and when using the 1.4.1 version, I get errors like

```
could not get type for name io.deepsense.deeplang.DOperation1To1 from any class loader
```

**Comment by jaroslaw-osmanski** [comment]
Thank you for submitting this bug report. We published 1.4.2 deeplang version to sonatype and seahorse-sdk-example master should build properly now.

**Comment by jaroslaw-osmanski** [comment]
Thank you for submitting this bug report. We published 1.4.2 deeplang version to sonatype and seahorse-sdk-example master should build properly now.
